Python/Password_Generator/password_generator.py

```diff
@@ -0,0 +1,40 @@
+import string
+import random
+
+length = int(input("Enter length of your password:\n"))
+
+DIGITS = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+UPPER_CASE = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H','I', 
+              'J', 'K', 'M', 'N', 'O', 'P', 'Q','R', 'S', 
+              'T', 'U', 'V', 'W', 'X', 'Y','Z']
+LOWER_CASE = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h',
+              'i', 'j', 'k', 'm', 'n', 'o', 'p', 'q',
+              'r', 's', 't', 'u', 'v', 'w', 'x', 'y',
+              'z']
+SYMBOLS = ['@', '#', '$', '%', '=', ':', '?', '.', '/', 
+           '|', '~', '>', '*', '(', ')', '<']
+
+#ALl the valid characters that can be used in a password
+VALID_CHARS = DIGITS+UPPER_CASE+LOWER_CASE+SYMBOLS
+
+#Adding atleast 1 character of all types of valid characters
+password_temp = random.choice(DIGITS)+ random.choice(UPPER_CASE)+ random.choice(LOWER_CASE)+ random.choice(SYMBOLS)
+
+if(length<8):
```

Comment:
Since you have added only 4 chars in the top,  the if condition will always be false right? 
So, what is the use of that if condition?